app/frontend/src/components/form/Step2.vue

```diff
@@ -74,6 +79,7 @@
               flat
               outlined
               solo
+              placeholder="xxx@xxx.com"
```

Comment:
Think we should have the placeholder not be "xxx" for... uh... reasons? Perhaps "person@example.com" might be better?

Reply:
Lol. :)
Took it from the pdf. Will run by Mina